CHANGELOG.md

```diff
@@ -1,12 +1,39 @@
 # Changelog
 
-## [0.0.1a2] - 2023-XX-XX
+## [0.0.1] - 2023-09-05
 
-### Highlights
+### Highlights:
+
+- Initial stable release of tstrait in PyPI https://pypi.org/project/tstrait/
+- Initial stable release of tstrait in conda-forge 
+
+### Documentation:
+
+- tstrait description in https://tskit.dev/software/
+
+### Contributors:
+
```

Comment:
Missing Gertjan Bisschop here

Reply:
Thank you for letting me know about it. I just included the people who made the commitment during the period, but from next time, I will definitely include everyone's name, as I wouldn't have been able to complete these commitments without everyone's support.